aspnetcore/includes/environmentVarableColon.md

```diff
@@ -1,4 +1 @@
-The `:` separator doesn't work with environment variable hierarchical keys on all platforms. `__`, the double underscore, is:
-
-* Supported by all platforms. For example, the `:` separator is not supported by [Bash](https://linuxhint.com/bash-environment-variables/), but `__` is.
-* Automatically replaced by a `:`
\ No newline at end of file
+The `:` separator doesn't work with environment variable hierarchical keys on all platforms. `__`, the double underscore, is supported by all platforms. For example, the `:` separator is not supported by [Bash](https://linuxhint.com/bash-environment-variables/), but `__` is.
```

Comment:
```suggestion
The `:` separator doesn't work with environment variable hierarchical keys on all platforms. For example, the `:` separator is not supported by [Bash](https://linuxhint.com/bash-environment-variables/). The double underscore, `__`, is:
* Supported by all platforms.
* Automatically replaced by a colon, `:`.
```
See the original version of this here: https://github.com/dotnet/AspNetCore.Docs/pull/11731/files